conf/messages.cy

```diff
@@ -423,8 +423,8 @@ checkYourAnswers.agents.whatCausedYouToMissTheDeadline.agent = Roedd rhywbeth ar
 # Appeal Confirmation page
 # ----------------------------------------------------------
 
-appealConfirmation.headingAndTitle.penalty = You have asked us to review a penalty
-agent.appealConfirmation.headingAndTitle.penalty = You have asked us to review your client’s penalty
+appealConfirmation.headingAndTitle.penalty = You have asked us to review your penalty
+agent.appealConfirmation.headingAndTitle.penalty = Rydych wedi gofyn i ni adolygu cosb
 appealConfirmation.headingAndTitle.penalties = You have asked us to review your penalties
 agent.appealConfirmation.headingAndTitle.penalties = You have asked us to review your client’s penalties
```

Comment:
Does this not need to change?

Reply:
If I am understanding correctly what you are referring to no it  doesn't, Linds said for agents it should be:
- Singular appeal `You have asked us to review a penalty`
- Multi appeal `You have asked us to review your client’s penalties`
We do not have the welsh yet for the later